vizro-core/docs/pages/user_guides/selectors.md

```diff
@@ -0,0 +1,45 @@
+# How to use selectors
+
+This guide shows you the differences in selectors for your dashboard. Selectors do not serve a function on their own, but they let you _select_ the input for other models, e.g. the [`Filter`][vizro.models.Filter] or the [`Parameter`][vizro.models.Parameter] model.
```

Comment:
Should `_select_` be changed to ``select``?

Reply:
It was supposed to be a highlight, as in why its a basically called slector. `_abc_` just makes things itallic.